### PR TITLE
Update `AbsoluteLayoutFlags` Functionality

### PR DESCRIPTION
**Pull request by brminnick**
This PR adds `.LayoutFlags()` funcitonality to the docs as introduced in https://github.com/CommunityToolkit/Maui.Markup/issues/215

### CODE REVIEW

**Review by bijington — APPROVED** [review]
At first I thought we needed to add some context around `,` vs `|` but I think that should just be discovered from the samples and usage. 

Good work @brminnick

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
I _almost_ merged this one... I guess this is still a draft until we merge the feature?

**Comment by bijington** [comment]
Yeah let's hold off until we release the feature

**Pull request merged**
